src/handlers/dns/DnsStreamHandler.h

```diff
@@ -114,6 +117,8 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
         : _dnsXactFromTimeUs("dns", {"xact", "out", "quantiles_us"}, "Quantiles of transaction timing (query/reply pairs) when host is client, in microseconds")
         , _dnsXactToTimeUs("dns", {"xact", "in", "quantiles_us"}, "Quantiles of transaction timing (query/reply pairs) when host is server, in microseconds")
         , _dns_qnameCard("dns", {"cardinality", "qname"}, "Cardinality of unique QNAMES, both ingress and egress")
+        , _dns_topGeoLocECS("dns", "top_geoloc", {"top_geoLoc_ecs"}, "Top GeoIP ECS locations")
```

Comment:
second argument should be "geo_loc" as the label for the item... to match https://github.com/ns1labs/pktvisor/blob/951979557ea3b6e78a849d4c9bdbcaeeab87dc61/src/handlers/net/NetStreamHandler.h#L109

Reply:
done